deployments/gpu-operator/templates/dcgm_exporter_config.yaml

```diff
@@ -0,0 +1,14 @@
+{{- if .Values.dcgmExporter.config }}
+{{- if and (.Values.dcgmExporter.config.create) (not (empty .Values.dcgmExporter.config.data)) }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Values.dcgmExporter.config.name }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "gpu-operator.labels" . | nindent 4 }}
+data:
+  dcgm-metrics.csv: |
+{{- .Values.dcgmExporter.config.data | nindent 4 }}
+{{- end }}
+{{- end }}
```

Comment:
add EOL

Reply:
Not sure I follow you on this comment. Can you explain?

Reply:
If you look at the diff of this file, you'll see a red icon added by GitHub which denotes a missing EOL character. You can fix this by adding an empty newline which will act as the EOL marker of your file

See [here](https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-a-source-file) for context